templates/docs/document.html

```diff
@@ -3,6 +3,8 @@
 {% block page_class %}docs{% endblock %}
 {% block content_class %}l-docs-wrapper{% endblock %}
 {% block page_title %}| {{ document.title }} {% endblock %}
+{% block meta_copydoc %}{% endblock meta_copydoc %}
+
```

Comment:
Taylor wants to know why the Blank Space?

Reply:
I guess I could remove the default one from the base template :shrug:

Reply:
To be clear, I'm suggesting nuking the blank line 7 of the diff, so there's just one blank line between {% endblock %} and the {% block content %}

Reply:
Done.